src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -377,33 +378,19 @@ def update_groups(project, seer_response, group_id_batch_filtered, group_hashes_
     )
 
 
-@metrics.wraps(f"{BACKFILL_NAME}.lookup_event_bulk", sample_rate=1.0)
-def lookup_group_data_stacktrace_bulk(
-    project: Project, rows: list[GroupEventRow]
-) -> dict[int, Event]:
-    project_id = project.id
-    node_id_to_group_data = {
-        Event.generate_node_id(project_id, event_id=row["event_id"]): (
-            row["event_id"],
-            row["group_id"],
-        )
-        for row in rows
-    }
-
-    groups_to_event = {}
-
+def _make_nodestore_call(project, node_keys):
     try:
         bulk_data = _retry_operation(
             nodestore.backend.get_multi,
-            list(node_id_to_group_data.keys()),
+            node_keys,
             retries=3,
             delay=2,
         )
     except (ServiceUnavailable, DeadlineExceeded) as e:
         extra = {
             "organization_id": project.organization.id,
             "project_id": project.id,
-            "group_data": json.dumps(rows),
+            "node_keys": json.dumps(node_keys),
```

Comment:
not logging rows here because i don't want to pass them in, node keys should be enough for us to debug

Reply:
(i don't believe this error has happened, either